libs/dkg/src/dkg_service.cpp

```diff
@@ -114,6 +114,7 @@ DkgService::DkgService(Endpoint &endpoint, ConstByteArray address, ConstByteArra
   , rpc_server_{endpoint_, SERVICE_DKG, CHANNEL_RPC}
   , rpc_client_{"dkg", endpoint_, SERVICE_DKG, CHANNEL_RPC}
   , state_machine_{std::make_shared<StateMachine>("dkg", State::BUILD_AEON_KEYS, ToString)}
+  , rbc_{endpoint_, address_, current_cabinet_, current_threshold_, *this}
```

Comment:
Just a note: as wojciech pointed out, need to be careful when refrencing this in the constructor of something - I think that if rbc took a ```DkgService dkg_service``` (it doesn't) it would copy construct DkgService partway through constructing it, which would likely cause a segfault.

I checked and this usage is safe, though.